blank-canvas-blocks/sass/blocks/_style.scss

```
@@ -0,0 +1,4 @@
+@import "button/style";
```

Looks like this hasn't been brought over yet so this needs to be removed or the button styles added.

```suggestion
@import "button/style";
```